### PR TITLE
Pull Request 🫡

### PR DESCRIPTION
**Pull request by shashankpandey04**

### CODE REVIEW

File: cogs/ActivityNotices.py
Comment (MikeyUsersREC) [inline cogs/ActivityNotices.py:1]: Fair enough way of doing this. Satisfies both LOA and RA prerequisites.

File: cogs/CustomCommands.py
Comment (MikeyUsersREC) [inline cogs/CustomCommands.py:1]: There's no need to revert my code here.
Eventually there was meant to be a feature to allow for multiple embeds in a single custom command message.
It was never planned, however it shouldn't be discarded in my opinion.

File: erm.py
Comment (MikeyUsersREC) [inline erm.py:1]: Line 612 will prevent tempban checks from even happening.
Raising an exception will stop the entire task, not just for that server.

File: events/on_message.py
Comment (MikeyUsersREC) [inline events/on_message.py:1]: No. I don't think you understand how RCE is meant to work.
It essentially tricks the bot into thinking that you ran a command, so if you want to autokick someone, make the bot think that you ran `punish i_imikey warning rdm --kick` as an alias.

**Review by MikeyUsersREC — CHANGES_REQUESTED** [review]
Changes required.

**Comment by shashankpandey04** [comment]
Added new command: `duty shifts`, which allows Management to view staff previous shifts in one go.

**Pull request closed without merging**